### ISSUE TITLE
settings.json is overwrited each time when admin app starts

### ISSUE THREAD

**Issue by i-erokhin** [simov/express-admin#102]
node version v4.3.1
OS X version 10.11.3

I just run 
```
$ admin ./conf 
``` 
and my settings.json (located in ./conf) rollback to first-run state.

**Comment by simov** [comment]
That's weird. I've never experienced such thing. The configuration logic only tries to add new tables or columns, it never deletes anyhitng.

**Comment by i-erokhin** [comment]
My custom settings.json contains only one table, but db contains many tables, maybe it`s the problem? I'm just trying to disable other tables, and each time the application adds these tables in my settings.json.  

Is this expected behavior?

**Comment by simov** [comment]
Yes, disabling them means you have to set `mainview:show:false` for each table you want to hide.

**Comment by i-erokhin** [comment]
Ok. What`s the minimal configuration for tables what i just want to hide? (It`s not covered by the docs)

**Comment by simov** [comment]
You can't hide the configuration that's been generated for you. You can only use it to configure your admin.

**Comment by i-erokhin** [comment]
Thank you very much.

**Comment by simov** [comment]
I understand. Currently your best option is to use the code folding feature of your editor. For example in Sublime Text it is: `Ctrl+K+2` to fold all to second indent. So using this feature you can fold to all table keys.

**Comment by ashnur** [comment]
I have the same issue, settings.json is overwritten on application start.

**Comment by flyingsky** [comment]
see this pull request https://github.com/simov/express-admin/pull/113
